src/Uno.UWP/Graphics/Display/DisplayInformation.iOS.cs

```diff
@@ -37,8 +37,10 @@ private void UpdateProperties()
 
 		private void UpdateLogicalProperties()
 		{
-			LogicalDpi = (float)(UIScreen.MainScreen.Scale * 100);
-			ResolutionScale = (ResolutionScale)(int)LogicalDpi;
+			// Scale of 1 is considered @1x, which is the equivalent of 96.0 or 100% for UWP.
+			// https://developer.apple.com/documentation/uikit/uiscreen/1617836-scale
+			LogicalDpi = (float)(UIScreen.MainScreen.Scale * 96.0f);
+			ResolutionScale = (ResolutionScale)(int)((LogicalDpi / 96.0) * 100);
```

Comment:
I would be mildly concerned about floating-point errors here - would be better to avoid multiplying and then dividing `MainScreen.Scale` by 96

Reply:
That's fair, will adjust.